### PR TITLE
Refactor: Remove wrapping view from withHover

### PR DESCRIPTION
**Pull request by tbergq**
Summary:Wrapping view caused malformed style when applied to components.
Instead inject the onMouseLeave and onMouseEnter through the HOC.

### CODE REVIEW

File: apps/graphql/package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "serverless-offline": "^4.9.4",
     "serverless-webpack": "^5.2.0",
     "uglifyjs-webpack-plugin": "^2.1.2",
-    "webpack": "^4.29.1"
+    "webpack": "^4.29.1",
+    "webpack-node-externals": "^1.7.2"
```
Comment (RobinCsl) [inline apps/graphql/package.json]: This is not really related, but okay :D

Comment (tbergq) [reply apps/graphql/package.json]: Maybe not, but it still gave eslint error
